bleep/src/main/scala/bleep/BleepCommandRemote.scala

```diff
@@ -0,0 +1,44 @@
+package bleep
+
+import bleep.internal.{BspClientDisplayProgress, MyBloopRifleLogger}
+import bleep.model.ProjectName
+import cats.effect.{ExitCode, IO}
+import ch.epfl.scala.bsp4j
+import ch.epfl.scala.bsp4j.BuildTargetIdentifier
+
+import java.util
+import scala.build.bloop.{BloopServer, BloopThreads}
+import scala.jdk.CollectionConverters._
+
+trait BleepCommandRemote extends BleepCommand {
+  def targetId(buildPaths: BuildPaths, name: ProjectName): bsp4j.BuildTargetIdentifier =
+    new bsp4j.BuildTargetIdentifier(buildPaths.buildDir.toFile.toURI.toASCIIString.stripSuffix("/") + "/?id=" + name.value)
+
+  def chosenTargets(started: Started): util.List[BuildTargetIdentifier] =
+    started.chosenProjects.map(projectName => targetId(started.buildPaths, projectName)).asJava
+
+  def runWithServer(started: Started, bloop: BloopServer): Unit
+
+  override final def run(): IO[ExitCode] =
+    runWithEnv { started: Started =>
+      val bloopRifleConfig = new BloopSetup(
+        "/usr/bin/java",
```

Comment:
$JAVA_HOME/bin/java, the we can fallback to this?

Reply:
Or even better https://github.com/oyvindberg/bleep/issues/1 . Up for grabs if you wanna try, otherwise I'll get to it in some days